control/controlclient/map.go

```diff
@@ -343,6 +352,73 @@ func undeltaPeers(mapRes *tailcfg.MapResponse, prev []*tailcfg.Node) {
 	mapRes.PeersRemoved = nil
 }
 
+// For extra defense-in-depth, when we're testing expired nodes we check
+// ControlTime against this 'epoch' (set to the approximate time that this code
+// was written) such that if control (or Headscale, etc.) sends a ControlTime
+// that's sufficiently far in the past, we can safely ignore it.
+var removeExpiredPeersEpoch = time.Unix(1673373066, 0)
```

Comment:
rename "remove expired peers" here and line 366/371 to something like "breakExpiredPeers" or "flagExpiredPeers" or "noteExpiredPeers" ?